addons/website/views/website_templates.xml

```diff
@@ -1178,7 +1148,7 @@
         <div class="oe_structure oe_structure_solo" id="oe_structure_header_magazine_1">
             <section class="s_text_block" data-snippet="s_text_block" data-name="Text">
                 <div class="container">
-                    <div class="row align-items-center">
+                    <div class="row align-items-center text-center">
```

Comment:
Don't add text alignment classes on the row but on the same element where the editor would add it (the p? the column? probably not the row)